src/meta/src/hummock/manager/versioning.rs

```diff
@@ -27,21 +27,30 @@ use risingwave_pb::hummock::{
     HummockVersionStats,
 };
 
+use crate::hummock::manager::worker::{HummockManagerEvent, HummockManagerEventSender};
 use crate::hummock::manager::{read_lock, write_lock};
 use crate::hummock::HummockManager;
 use crate::storage::MetaStore;
 
 /// `HummockVersionSafePoint` prevents hummock versions GE than it from being GC.
 /// It's used by meta node itself to temporarily pin versions.
-#[derive(Clone, PartialEq, Eq)]
 pub struct HummockVersionSafePoint {
```

Comment:
Better add a metric for the min safe point so that if min pinned id is stuck we know whether it is caused by backup or CN version pin.